lib/money/money.rb

```diff
@@ -215,7 +215,7 @@ def to_liquid
   end
 
   def to_json(options = {})
-    to_s
+    "\"#{to_s}\""
```

Comment:
If changing the representation at this time is not what we want, and the idea is to keep the core representation as a "string", then I'd say something that would achieve something similar yet provide more meaning to the code deferring the serialization logic to an existing layer that understand how to dump a string as json.

Eg.:
```ruby
def to_json(options = {})
  to_s.to_json
end
```

What worries me here is the options are discarded and I don't know enough about rails serialization to know if this comes from Rails or a param to the method that shouldn't have existed in the first place. It does not prevent the changes proposed here, but raises to me question as to if this new method definition is still broken in some way.